### ISSUE TITLE
npm audit fix fails with severity high because of dependency markdown-to-jsx   

### ISSUE THREAD

**Issue by wschaef** [storybookjs/storybook#10871]
**Describe the bug**
Result of **npm audit**

```
  High            Cross-Site Scripting                                          

  Package         markdown-to-jsx                                               

  Patched in      No patch available                                            

  Dependency of   @storybook/html [dev]                                         

  Path            @storybook/html > @storybook/core > @storybook/ui >           
                  markdown-to-jsx                                               

  More info       https://npmjs.com/advisories/1219      
```

**To Reproduce**
1. npm install @storybook/html --save-dev
2. npm audit fix
3. See error

**Additional context**
Even if it is a static content deployment at the end, it is still relevant if any other systems are available by the same domain.
Currently this issue is a show stopper for all public facing installations

**Comment by shilman** [comment]
Yowza!! I just released https://github.com/storybookjs/storybook/releases/tag/v6.0.0-beta.13 containing PR #10873 that references this issue. Upgrade today to try it out!

You can find this prerelease on the `@next` NPM tag.

Closing this issue. Please re-open if you think there's still more to do.

**Comment by shilman** [comment]
Yippee!! I just released https://github.com/storybookjs/storybook/releases/tag/v5.3.19 containing PR #10873 that references this issue. Upgrade today to try it out!

**Comment by merlinpatt** [comment]
I'm still seeing this as an issue in 5.3.19 even after uninstalling and reinstalling @storybook/react. Both @storybook/ui and @storybook/components depends on this and it's still causing a vulnerability.

**Comment by shilman** [comment]
Looks like `markdown-to-jsx` is on it: https://github.com/probablyup/markdown-to-jsx/issues/306

Will upgrade when ready

**Comment by msvivianso** [comment]
Looks like the fix in v6.11.4 from `markdown-to-jx` has been whitelisted from the vulnerability advisory.

**Comment by shilman** [comment]
@msvivianso thanks so much for letting me know. 🙏  closing!
